server/config.py

```diff
@@ -35,6 +35,18 @@ def __init__(self):
         Change default values here.
         """
         self.CONFIGURATION_REFRESH_TIME = 300
+        self.LISTEN = [
+            {
+                "address": "",
+                "port": 8001,
+                "protocol": "QDataStreamProtocol",
```

Comment:
We can rename these keys to whatever. Maybe caps make sense to be consistent with the rest of the config?

Reply:
Would these be set with dot notation in the yaml? It might be good to be consistent

Reply:
Not sure what you mean by 'dot notation'. In yaml it looks like this:
```yaml
LISTEN:
  - address: ""
    port: 8001
    protocol: QDataStreamProtocol

  - address: ""
    port: 8002
    protocol: SimpleJsonProtocol
```

Reply:
Yeah sorry meant to say they would be their own fields.

Then yeah looking at this I might say it would be good to be consistent.

Although I am sure others might say otherwise to indicate it is a sub field. But I think the indent solves that